backend/routes/api.php

```diff
@@ -99,6 +99,7 @@
     TemplateController::class,
     "userTemplatesIndex",
 ]);
+Route::get("template/{journey}/user", [JourneyUserController::class, "index"]);
```

Comment:
because we need to fetch the creator of a template and I want to keep that route separate from the one to fetch journey members so that I could change it more easily in the future

Reply:
I was told that the creator is not saved to the store so it needs to be fetched separately again